mags/python/mags/planning/graph.py

```diff
@@ -59,7 +59,7 @@ def __lt__(self, other):
 class Edge(UserList):
     """
     Class that represents an edge in the graph. This is a line segment or arc between two nodes.
-    NOTE: Surfing edges are line segments, while hugging edges are arcs.
+    NOTE: Surfing edges are line segments, while hugging edges are arcs. Tangent edges are tangent line segments from a point to a circle.
```

Comment:
This comment doesn't really make sense here because tangent edges are a subset of surfing edges and there is no distinction between them in the edge class.